fcrepo-indexing-triplestore/src/test/java/org/fcrepo/camel/indexing/triplestore/RouteTest.java

```diff
@@ -87,7 +86,7 @@ public static void beforeClass() {
     }
 
     private static Map<String, Object> createEvent(final String identifier, final List<String> eventTypes,
-                                                   final List<String> resourceTypes) {
+        final List<String> resourceTypes) {
```

Comment:
Do not change formatting of existing files.

Reply:
Fixed with https://github.com/docuteam/fcrepo-camel-toolbox/pull/1/commits/1af770afb19525576854e0bd4e85099aa5090211